<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="tabMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>371</height>
     </rect>
    </property>
    <widget class="QFrame" name="MainFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>531</width>
       <height>371</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="LblMainTitle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Another Media Player</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="BtnPlay">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnPause">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnStop">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </widget>
     <widget class="QDial" name="DialVolume">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="LineEditFileName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>391</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>/home/faisal/Downloads/WaveFiles/familyguy1.mp3</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>391</width>
        <height>29</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txtTrackInfo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(85, 255, 0);</string>
      </property>
     </widget>
     <widget class="QListView" name="ListSongs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>391</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnFileBowser">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>150</y>
        <width>97</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
